referenceframe/transformable.go

```diff
@@ -236,15 +254,14 @@ func (gF *GeometriesInFrame) Transform(tf *PoseInFrame) Transformable {
 
 // NewGeometriesInFrame generates a new GeometriesInFrame.
 func NewGeometriesInFrame(frame string, geometries []spatialmath.Geometry) *GeometriesInFrame {
-	geometriesMap := map[string]spatialmath.Geometry{}
-	for _, geometry := range geometries {
-		geometriesMap[geometry.Label()] = geometry
+	nameIndexMap := make(map[string]int)
```

Comment:
Given the slice tricks probably won't happen in the same way, does the change from `map[string]Geometry` need to persist?

Reply:
it doesn't, but as @biotinker pointed out in a different comment, having a `map[string]Geometry` was redundant

Reply:
Yeah, I'm good with the rationale posted elsewhere.